src/pseudo/Engine.java

```diff
@@ -6,7 +6,7 @@
 public class Engine {
 	//const values
 	
-	public static final byte ZERO = 0;
+	public static final byte ZERO = 0;   // <-- do we really need to make constants for that?
```

Comment:
Maybe not, I just prefer when all existing flags are defined as constants. But no difference for me in this case.